ecosystem/ecosystem.html

```diff
@@ -61,29 +63,32 @@ <h1>
       </nav> -->
 
       <!-- START CONTENT -->
-
-      <div class="row ecosystem-cards-wrapper">
-        {% assign ecosystem = site.ecosystem  | sort_natural: "title" %}
-
-        {% for item in ecosystem %}
-          <div class="col-md-6 ecosystem-card-wrapper" data-categories="{{ item.category | join: "," }}">
-            <div class="card ecosystem-card">
-              <a href="{{ item.link }}" target="_blank">
-                <div class="card-body {{ item.logo-class }} {% if item.experimental %}experimental{% endif %}">
-                  <div class="ecosystem-card-title-container">
-                    <h4>{{ item.title }}</h4>
-                      {% if item.experimental %}
-                        <div class="experimental-badge">
-                          Experimental
-                        </div>
-                      {% endif %}
+      <div id="ecosystem-index-cards">
+        <div class="list row ecosystem-cards-wrapper">
+          {% assign ecosystem = site.ecosystem  | sort_natural: "title" %}
+
+          {% for item in ecosystem %}
+            <div class="col-md-6 ecosystem-card-wrapper" data-categories="{{ item.category | join: "," }}">
+              <div class="card ecosystem-card">
+                <a href="{{ item.link }}" target="_blank">
+                  <div class="card-body {{ item.logo-class }} {% if item.experimental %}experimental{% endif %}">
+                    <div class="ecosystem-card-title-container">
+                      <h4>{{ item.title }}</h4>
+                        {% if item.experimental %}
+                          <div class="experimental-badge">
+                            Experimental
+                          </div>
+                        {% endif %}
+                    </div>
+                    <p class="card-summary">{{ item.summary }}</p>
```

Comment:
It looks like a lot of this is changed from the `ecosystem-stars-final` branch. It doesn't have any of the new data attributes we added. Can you make sure you're branching off of `ecosystem-stars-final`?